.template/addons/nginx/config/nginx/app.conf.template.tt

```diff
@@ -0,0 +1,113 @@
+worker_processes auto;  ## Default: 1
+
+events {
+  worker_connections 1024;  ## Default: 1024
+}
+
+http {
+  include /etc/nginx/mime.types;
+
+  gzip on;
+  gzip_comp_level 5;
+  gzip_min_length 256;
+  gzip_proxied any;
+  gzip_vary on;
+  gzip_types
+    application/atom+xml
+    application/geo+json
+    application/javascript
+    application/json
+    application/ld+json
+    application/manifest+json
+    application/rdf+xml
+    application/rss+xml
+    application/vnd.ms-fontobject
+    application/wasm
+    application/x-web-app-manifest+json
+    application/xhtml+xml
+    application/xml
+    font/otf
+    image/bmp
+    image/svg+xml
+    text/cache-manifest
+    text/calendar
+    text/css
+    text/javascript
+    text/markdown
+    text/plain
+    text/vcard
+    text/vnd.rim.location.xloc
+    text/vtt
+    text/x-component
+    text/x-cross-domain-policy;
+
+  upstream app {
+    server localhost:3000;
+  }
+
+  server {
+    server_name _;
+
+    root /<%= APP_NAME %>/public;
+
+    proxy_set_header  X-Real-IP  $remote_addr;
+    proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
```suggestion
    proxy_set_header X-Real-IP  $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Remove unnecessary 2 spaces.

Reply:
@byhbt could you explain more about `the alignment of the space is not consistent in this files` 🤔  at which line the alignment isn't consistent

Reply:
```
      proxy_buffer_size          12k;
      proxy_buffers              8 12k;
      proxy_busy_buffers_size    24k;
```
I think we can update it to: 😄 

```
      proxy_buffer_size 12k;
      proxy_buffers 8 12k;
      proxy_busy_buffers_size 24k;
```